bouncer/scripts/test/redirect-test.js

```diff
@@ -110,6 +111,24 @@ describe('redirect()', function () {
     assert.equal(console.error.called, true);
   });
 
+  it('calls chrome.runtime.sendMessage correctly', function () {
+    window.chrome = {
+      runtime: {
+        sendMessage: sinon.stub()
+      }
+    };
+
+    redirect(function () {});
+
+    assert.equal(window.chrome.runtime.sendMessage.calledOnce, true);
```

Comment:
@seanh - Sinon has a concept called [matchers](http://sinonjs.org/docs/#sinon-match-api) which allow you to provide arguments to `assert.calledWith` that are not the exact object you want to match but instead a predicate that tests whether the argument matches some condition.